homeassistant/components/samsungtv/__init__.py

```diff
@@ -269,7 +269,7 @@ async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
         en_reg = er.async_get(hass)
         en_reg.async_clear_config_entry(config_entry.entry_id)
 
-        version = config_entry.version = 2
+        hass.config_entries.async_update_entry(config_entry, version=2)
```

Comment:
Just a thought... this removes the update of the version variable so the debug log below is now incorrect.

It results in "Migration to version 1 successful" instead of "Migration to version 2 successful"

Reply:
Good catch. Fixed in https://github.com/home-assistant/core/pull/110484